app/units/units.css

```diff
@@ -111,6 +118,13 @@ span.icon {
     color: #DD432B;
     font-size: 35px;
     z-index: 2;
+    cursor: pointer; /* Add cursor pointer */
+    user-select: none; /* Make it unselectable */
+}
+
+span.icon:hover {
```

Comment:
We only want the filter icon to be black on hover, however your change also adds the hover effect to the search icon. Please fix this issue.